### DISCUSSION TITLE
Question about the translation process

### DISCUSSION THREAD

**Discussion by ubanning** [Dadangdut33/Speech-Translate#16] [Q&A]
Hello, how are you? Thanks for the project.
Could you let me know if you use any method when translating using Google Translate to improve the context? Or is it a simple line break? I'm trying to translate SRT files, but I'm having trouble maintaining the context using just a line break. I've already tried to use characters to separate sentences, but Google Translate sometimes removes these characters and so I end up losing the reference to the caption number where the translation should be placed. In case your project does not use any method to improve the context of the translated subtitle, we could think of a strategy together. Thanks :)

**Comment by Dadangdut33** [comment]
Good questions, it is something that should be improved. Can you give me more example on the translation that is lost and also what do you suggest in mind?

**Comment by ubanning** [comment]
Hello, I really don't have an idea, I've already tried all the alternatives that came to my mind. All alternatives, at some point, the translator modifies my reference marker to the subtitle number.
I've tried using several alternatives, such as:
(1) Hello, (2) how are you?
Hello, :: how are you?
Hello, // how are you?
Hello, || how are you?

I don't know if you understand my idea, if you don't, let me know.
But basically I would like to translate the subtitle, and I need the translator to understand that it is a single text, so that the context is not lost, because if I translate subtitle by subtitle, the quality of the translation drops a lot, because the translator loses context. So somehow I need to leave the text together so that after the translation I can have some reference to generate a new translated SRT file.
